moped-editor/src/views/projects/projectView/ProjectContractsTable.js

```diff
@@ -198,6 +216,9 @@ const ProjectContractsTable = () => {
 
             // Remove unneeded variable
             delete updateContractData.__typename;
+            updateContractData.contract_amount = Number(
+              newData.contract_amount
+            );
```

Comment:
this is what the funding amount table has that the contract amount didnt -- what changes the "" to a 0 so it works in the mutation that is looking for an Int!

Reply:
ah hah. ok, so if we're defaulting the amount to $0 in the UI, i think you should also set the DM column default to `0`.

Reply:
updated!